### ISSUE TITLE
will there be a similar/updated sample tensorflow-recommendation-ncf since wals is deprecated in Tensorflow 2.0+?

### ISSUE THREAD

**Issue by victusfate** [GoogleCloudPlatform/tensorflow-recommendation-wals#15]
reviewing https://github.com/tensorflow/models/tree/08bb9eb5ad79e6bceffc71aeea6af809cc78694b/official/recommendation for how to get started

**Comment by victusfate** [comment]
I was able to dig around and find a very similar approach in the tutorials
https://developers.google.com/machine-learning/recommendation/dnn/softmax

(props to whoever put those tutorials together was helpful getting another look at the approach and tradeoffs)

**Comment by victusfate** [comment]
note for anyone else looking for example apps I bumped into a couple by Nvidia
https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/Recommendation/NCF
https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow/Recommendation/NCF

cool overview for intel's analytics zoo (office depot) across models MF, ncf, wide & deep, and session recommender
https://software.intel.com/en-us/articles/real-time-product-recommendations-for-office-depot-using-apache-spark-and-analytics-zoo-on


it looks like Google's official example leverages apache spark's als model
https://cloud.google.com/solutions/recommendations-using-machine-learning-on-compute-engine
